.github/workflows/release.yml

```
@@ -44,17 +44,41 @@ jobs:
 
       - name: Build RPM package for EL8
         id: rpm_build_el8
-        uses: bocekm/rpmbuild@el8
+        uses: oamg/rpmbuild@el8
         with:
           spec_path: "packaging/convert2rhel.spec"
 
       - name: Upload EL8 RPM as release asset
         id: upload_release_asset_el8
         uses: actions/upload-release-asset@v1
         env:
-            GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         with:
-            upload_url: ${{ github.event.release.upload_url }}
-            asset_path: ${{ steps.rpm_build_el8.outputs.rpm_path }}
-            asset_name: ${{ steps.rpm_build_el8.outputs.rpm_name }}
-            asset_content_type: ${{ steps.rpm_build_el8.outputs.content_type }}
+          upload_url: ${{ github.event.release.upload_url }}
+          asset_path: ${{ steps.rpm_build_el8.outputs.rpm_path }}
+          asset_name: ${{ steps.rpm_build_el8.outputs.rpm_name }}
+          asset_content_type: ${{ steps.rpm_build_el8.outputs.content_type }}
+
+  build_el9_rpm:
+    name: Build EL9 RPM
```

I think skip for now as we don't have it setup yet so it will fail
```suggestion
  build_el9_rpm:
    if: false
    name: Build EL9 RPM
```